GitCommands/Git/GitModule.cs

```diff
@@ -1311,12 +1311,14 @@ public bool ExistsMergeCommit(string startRev, string endRev)
 
             string revisions = RunGitCmd("rev-list --parents --no-walk " + startRev + ".." + endRev);
             string[] revisionsTab = revisions.Split('\n');
-            Func<string, bool> ex = (string parents) =>
-                {
-                    string[] tab = parents.Split(' ');
-                    return tab.Length > 2 && tab.All(parent => GitRevision.Sha1HashRegex.IsMatch(parent));
-                };
-            return revisionsTab.Any(ex);
+
+            bool IsTwoSha1Hashes(string parents)
```

Comment:
:bulb: We typically place local functions at the end of the method, after the last `return`. For a `void`-returning method, we would include an explicit `return` statement prior to the first local function.

Reply:
I do this too (though I hadn't considered the `return` in `void` methods, which is nice). However moving them will increase the diff considerably which I've had push back on in previous reviews. Can make this change if needed.

Reply:
If this change is done as a separate PR - we can possibly get it quickly even though the diff may be destructive.
We have to be conscious though of other pending PRs - some of them are quite large (#3693 #4499 etc) and rebasing them is a royal pain